lib/ansible/plugins/doc_fragments/vmware_rest_client.py

```diff
@@ -33,6 +33,13 @@ class ModuleDocFragment(object):
     - If the value is not specified in the task, the value of environment variable C(VMWARE_VALIDATE_CERTS) will be used instead.
     type: bool
     default: yes
+  port:
+    description:
+    - The port number of the vSphere vCenter.
+    - If the value is not specified in the task, the value of environment variable C(VMWARE_PORT) will be used instead.
```

Comment:
```suggestion
    - Environment variable support added in Ansible 2.10.
```

Reply:
There is already the `version_added: '2.10'`, this seems to be a bit redundant.